README.md

```diff
@@ -8,6 +8,8 @@ You can install Goby [here](https://chrome.google.com/webstore/detail/goby/jnkel
 
 ## Run your own node
 
+Install redis, e.g. sudo apt-get install redis
```

Comment:
Hi, @RightSexyOrc. Thanks for your PR. Would you like to make it more detailed?

Reply:
Hi, @RightSexyOrc . We removed Redis dependency.

Reply:
Oh, ok good. Sorry, didn't notice your Jan 2 query somehow. Thank yee